web/templates/draft_table_body.hbs

```diff
@@ -6,10 +6,26 @@
                 <div class="exit">
                     <span class="exit-sign">&times;</span>
                 </div>
-                <div class="removed-drafts">
-                    {{t "Drafts are not synced to other devices and browsers." }}
-                    <br />
-                    {{#tr}}Drafts older than <strong>{draft_lifetime}</strong> days are automatically removed.{{/tr}}
+                <div class="header-body">
+                    <div class="removed-drafts">
+                        {{t "Drafts are not synced to other devices and browsers." }}
+                        <br />
+                        {{#tr}}Drafts older than <strong>{draft_lifetime}</strong> days are automatically removed.{{/tr}}
+                    </div>
+                    <div class="delete-drafts-group">
+                        <div class="delete-drafts-tooltip-wrapper">
+                            <button class="button small rounded delete-selected-drafts-button" type="button" disabled>
+                                <div class="fa fa-trash-o fa-lg" aria-hidden="true"></div>
+                            </button>
+                        </div>
+                        <button class="button small rounded select-drafts-button" type="button">
```

Comment:
I'm not feeling good about this use of a `<button>` with an embedded `<label>` wrapping the actual control--the `<input>`--which then gets an `aria-hidden` attribute. I understand that we do some custom checkbox things with the `<span>`, but those are not going to be as accessible as the native checkbox controls.

As I noted [on CZO](https://chat.zulip.org/#narrow/stream/101-design/topic/bulk.20delete.20drafts.20.2319943/near/1585514), the button pattern is causing some unpredictable behavior for keyboard navigation as well.

Is there a reason to group these elements under a button? Especially because a properly associated `<label>` element will also function to activate the input (though, again, that might be short-circuited by the use of the inner `<span>` for displaying that control)?